### ISSUE TITLE
Sidecar pattern supported?

### ISSUE THREAD

**Issue by darkl0rd** [mthenw/frontail#212]
Maybe this is already possible and I simply don’t know how to configure it.

I am planning on exposing logs for a (dockerized) application through the web, but I don’t want to modify the image of the application. I stumbled upon this project and  my thought was to run this as a separate container which would display the logs from said application. However even though stdin is supported, I have no idea how I could have this container attach to the stdout/stderr of my other container.

TL;DR: I want to run this program as a separate container (sidecar) and have it expose the logs my application spits out on stderr/stdout.
